srcpkgs/xorg-server/template

```diff
@@ -34,7 +34,7 @@ build_options="elogind"
 desc_option_elogind="Rootless Xorg support with elogind"
 
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
I wonder if this case statement could be removed altogether. All officially supported arches are included, and ppc too. Is this important for some obscure arch?

Reply:
It excludes MIPS, right?

Reply:
yeah, mips and other arm* (`./xbps-src -h` gives a list)

Reply:
What I meant is that MIPS is also officially supported (at least that's what it says at voidlinux.org), so we have to consider that when building these packages.

Reply:
I don't think void builds packages for MIPS... also MIPS as an architecture is going the way of the dodo thanks to RISCV

Reply:
See http://repo.voidlinux.de by @pullmoll

Reply:
Technically, gallium-pipe is hardware agnostic and will support the obscure arches. It's not very efficient, though, but maybe there will be a better software renderer?

Reply:
@xtraeme do you think keeping this check for MIPS is the best way of doing it?